src/inputs/Rate.tsx

```diff
@@ -4,14 +4,24 @@ import { EMPTY_FIELD } from '@mighty-justice/utils';
 
 import * as Antd from 'antd';
```

Comment:
I've replicate how the RadioGroup is structured because `class Rate extends Component<any>` scared me 😱 .

But I might be wrong ...